src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28714.cs

```diff
@@ -0,0 +1,32 @@
+﻿#if TEST_FAILS_ON_WINDOWS // The transparent background color is not working on Windows, refer to https://github.com/microsoft/microsoft-ui-xaml/issues/6527
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+public class Issue28714 : _IssuesUITest
+{
+	public override string Issue => "[iOS] WebView BackgroundColor is not setting correctly";
+
+	public Issue28714(TestDevice device)
+	: base(device)
+	{ }
+
+	[Test, Order(1)]
+	[Category(UITestCategories.WebView)]
+	public void VerifyWebViewBackgroundColor()
```

Comment:
Pending snapshot on some platforms. Available in the latest build.

Example, Mac:
![image](https://github.com/user-attachments/assets/4ab81ebe-99be-4cde-ad6d-74457ab332e5)

Could you commit the images?

Reply:
@jsuarezruiz, the pending Mac snapshots have been added.